src/Supervisor.cpp

```diff
@@ -1088,4 +1088,93 @@ ZunResult Supervisor::PlayAudio(char *path)
     return ZUN_SUCCESS;
 }
 #pragma optimize("", on)
+
+// u32 g_ControllerData[32];
+DIFFABLE_STATIC_ARRAY(u32, 32, g_ControllerData)
+
+#pragma optimize("s", on)
+u8 *th06::Controller::GetControllerState()
+// this is for rebinding keys
+{
+    // DWORD_PTR cookie;
+    MMRESULT MVar1;
+    HRESULT HVar2;
+    u32 *piVar3;
+    int i;
+    u32 puVar3;
+    joyinfoex_tag *joyinfo_ptr;
+    u32 *puVar4;
+    int local_retryCount;
+    DIJOYSTATE2 local_15c;
+    int local_44;
+    u32 local_40;
+    u32 local_3c;
+    joyinfoex_tag joyinfo;
```

Comment:
```suggestion
    JOYINFOEX joyinfo;
```

The `_tag`-suffixed structs from windows shouldn't be used.